src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -834,14 +835,18 @@ def finalize_testcase(testcase_id, last_crash_result_dict, flaky_stack=False):
 
 def utask_postprocess(output):
   """Postprocess in a trusted bot."""
+  testcase_id = output.uworker_input.testcase_id
```

Comment:
let's just reuse output.uworker_input.testcase_id, I think it's a little safer when it's clear where it came from.